packages/components/date-picker/src/date-picker-com/panel-date-pick.vue

```diff
@@ -249,7 +249,6 @@ const { shortcuts, disabledDate, cellClassName, defaultTime } = pickerBase.props
 const defaultValue = toRef(pickerBase.props, 'defaultValue')
 
 const currentViewRef = ref<{ focus: () => void }>()
-
 const innerDate = ref(dayjs().locale(lang.value))
```

Comment:
`panel-date-pick.vue` and `panel-month-range.vue` are not involved in this function and do not need to be modified.

Reply:
Have Reverted

Reply:
It seems that no new commits are seen.